packages/period-selector-dialog/.gitignore

```diff
@@ -0,0 +1 @@
+!css/
```

Comment:
Can you investigate it would be possible to instead remove the css/ from the root .gitignore, instead of adding this .gitignore?  I'm suspecting that the only generated css directory is in the core build directory, and build directories are already ignored.